index.html

```diff
@@ -121,6 +146,57 @@ <h2>Project name goes here</h2>
                 </div>
             </div>
         </section>
+        <!-- works desktop -->
+        <section class="grid">
+            <div class="grid-1">
+                one
```

Comment:
- [ ] Kindly remove these numbers from the website to match the design in Figma 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/213870763-323529ab-c1a3-4026-8d4d-9ea2b4320ea9.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/213870782-180d372e-2553-4429-b289-b4ef548c4499.png)